modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -1504,7 +1524,8 @@ private String _getObjectEntryNotificationQueueEntryBody(
 				LanguageUtil.getLanguageId(LocaleUtil.US),
 				_portal.getPortalURL(
 					ServiceContextThreadLocal.getServiceContext(
```

Comment:
Hi @victorKammerer , can we use the variable `serviceContext` here?

Reply:
Hey @linolaoj, we do!